server/graphql/common/transactions.ts

```diff
@@ -89,14 +102,34 @@ export const canRefund = async (transaction, _, req): Promise<boolean> => {
   if (transaction.type !== TransactionTypes.CREDIT || transaction.OrderId === null) {
     return false;
   }
-  return remoteUserMeetsOneCondition(req, transaction, [isRoot, isPayeeHostAdmin]);
+  const timeLimit = moment().subtract(30, 'd');
+  const createdAtMoment = moment(transaction.createdAt);
+  const transactionIsOlderThanThirtyDays = createdAtMoment < timeLimit;
+  if (transactionIsOlderThanThirtyDays) {
+    return remoteUserMeetsOneCondition(req, transaction, [isRoot, isPayeeHostAdmin]);
+  } else {
+    return remoteUserMeetsOneCondition(req, transaction, [isRoot, isPayeeCollectiveAdmin]);
+  }
```

Comment:
This condition is problematic:
- If transaction is older than 30 days, only host admins can refund (good)
- Otherwise only collective admins can refund (we want host admins there too)